client/src/pages/admin-dashboard/index.jsx

```diff
@@ -1,7 +1,32 @@
-import UserManagement from "../../components/user-management-component/UserManagement";
+import PropTypes from "prop-types";
+import UserManagement from "../../components/user-management-component";
+import UserType from "../../prop-types/UserType";
 
-function AdminDashboard() {
-  return <UserManagement />;
+function AdminDashboard({ managedUsers, setManagedUsers }) {
+  const processedManagedUsers = managedUsers.map((user) => ({
+    ...user,
+    id: user.userID,
+    displayName: `${user.firstName} ${user.lastName}`,
+  }));
+
+  const removeUser = (userID) => {
```

Comment:
Do we have a commented-out API call to the backend to get all users? I know you have some mock users for temporary use while the Users backend is in limbo

Reply:
Hi Jordan, we don't have an endpoint to get all the users yet, not one that I was able to find.